### DISCUSSION TITLE
Nested OrderedList styling

### DISCUSSION THREAD

**Discussion by schostac** [ueberdosis/tiptap#636] [General Questions]
I'm using `OrderedList` and `ListItem` as in the example https://tiptap.scrumpy.io/. 
I wanted to make it nested like ` 1.1.2.`. So, I added some css:

```
ol {
  counter-reset: item;
}
li {
  display: block
}
li:before {
  content: counters(item, ".") ". ";
  counter-increment: item;
} 
```

However, paragraphs  appear under numbers like this
```
1.
   Some text
   1.1. 
       Some other text
```
Instead, I need 
```
1. Some text
    1.1. Some other text
```
How could I achieve that?

**Comment by hugh-sun-everlaw** [comment]
@schostac I ran into this also. Instead of using `li::before` in your css, use `li::marker`, then it should work I think!
